Makefile

```diff
@@ -1,12 +1,12 @@
-install-test:
```

Comment:
I'm dropping that one as relying on composer to install sass-spec is better: it ensures the right commit of sass-spec is used instead of taking the last one at the time you ran `make install-test`.
Thus, `make install-test` was probably broken if `sass-spec` is already installed.